dev-infrastructure/configurations/mgmt-nsp.tmpl.bicepparam

```diff
@@ -15,3 +15,5 @@ param aksKeyVaultName = '{{ .mgmt.aks.etcd.kvName }}'
 
 param mgmtNSPName = '{{ .mgmt.nsp.name }}'
 param mgmtNSPAccessMode = '{{ .mgmt.nsp.accessMode }}'
+
+param serviceClusterSubscriptionId = ''
```

Comment:
you need to use the __xxx__ syntax instead of an empty string.

https://github.com/Azure/ARO-HCP/blob/main/docs/bicep.md#output-templates-and-output-chaining